hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -56,10 +56,15 @@ public class MapConfig implements IdentifiedDataSerializable, NamedConfig {
      */
     public static final int MAX_BACKUP_COUNT = IPartition.MAX_BACKUP_COUNT;
 
+    /**
+     * The number of Time to Live that represents infinitive.
+     */
+    public static final int INFINITE_TTL_SECONDS = 0;
```

Comment:
Looks weird that infinity is represented as `0`. A better name might be `DISABLED_TTL_SECONDS`.

Reply:
Done